frontend/app/work_packages/controllers/work-package-details-controller.js

```diff
@@ -113,6 +113,8 @@ module.exports = function($scope,
     // watchers
     $scope.watchers = workPackage.embedded.watchers;
 
+    $scope.showErbPagePath = PathHelper.staticWorkPackagePath($scope.workPackage.props.id);
```

Comment:
@ulferts for consistency's sake, perhaps we can call this `showStaticPagePath`? Otherwise, fine.